sdk/python/feast/feature_logging.py

```diff
@@ -87,9 +86,14 @@ def get_schema(self, registry: "Registry") -> pa.Schema:
                     join_key = projection.join_key_map.get(
                         entity.join_key, entity.join_key
                     )
-                    fields[join_key] = FEAST_TYPE_TO_ARROW_TYPE[
-                        from_value_type(entity.value_type)
-                    ]
+                    entity_columns = list(
```

Comment:
can't we just iterate over entity columns in this for loop? seems like it has all information that we need for the schema generation.